sdks/python/apache_beam/runners/direct/transform_evaluator.py

```diff
@@ -191,8 +194,6 @@ def __init__(self, evaluation_context, applied_ptransform,
     self._execution_context = evaluation_context.get_execution_context(
         applied_ptransform)
     self.scoped_metrics_container = scoped_metrics_container
-    with scoped_metrics_container:
-      self.start_bundle()
```

Comment:
<!--new_thread; commit:09f68159df28b2c83a6be1643984c0e28507989b; resolved:1-->
Thanks for this change.